ee/fleetd-chrome/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "async-mutex": "^0.5.0",
     "dotenv": "^16.0.3",
+    "sass": "^1.83.4",
```

Comment:
As with the main fleet codebase, migrating from `node-sass` to `sass` in order to compile correctly on modern Node versions.

Reply:
I don't think this is even being used, but that too can be handled in a separate ticket.